packages/ui/src/views/repo/repo-layout/index.tsx

```diff
@@ -31,34 +37,39 @@ export const RepoLayout = ({ useTranslationStore }: { useTranslationStore: () =>
 
   return (
     <>
-      <SandboxLayout.SubHeader className="overflow-hidden">
-        <Tabs variant="navigation" value={activeTab}>
-          <TabsList>
-            <NavLink to={RepoTabsKeys.SUMMERY}>
-              <TabsTrigger value="summary">{t('views:repos.summary', 'Summary')}</TabsTrigger>
-            </NavLink>
-            <NavLink to={RepoTabsKeys.CODE}>
-              <TabsTrigger value="code">{t('views:repos.files', 'Files')}</TabsTrigger>
-            </NavLink>
-            <NavLink to={RepoTabsKeys.PIPELINES}>
-              <TabsTrigger value="pipelines">{t('views:repos.pipelines', 'Pipelines')}</TabsTrigger>
-            </NavLink>
-            <NavLink to={RepoTabsKeys.COMMITS}>
-              <TabsTrigger value="commits">{t('views:repos.commits', 'Commits')}</TabsTrigger>
-            </NavLink>
-            <NavLink to={RepoTabsKeys.PULLS}>
-              <TabsTrigger value="pulls">{t('views:repos.pull-requests', 'Pull Requests')}</TabsTrigger>
-            </NavLink>
-            <NavLink to={RepoTabsKeys.BRANCHES}>
-              <TabsTrigger value="branches">{t('views:repos.branches', 'Branches')}</TabsTrigger>
-            </NavLink>
-            <NavLink to={RepoTabsKeys.SETTINGS}>
-              <TabsTrigger value="settings">{t('views:repos.settings', 'Settings')}</TabsTrigger>
-            </NavLink>
-          </TabsList>
-        </Tabs>
-      </SandboxLayout.SubHeader>
-      <Outlet />
+      <div className="bg-background-1 sticky top-0 z-40">
+        {breadcrumbs}
```

Comment:
not a fan of breadcrumbs being passed in entirely as a single component prop.
we need to re-evaluate breadcrumbs as discussed here: https://gist.github.com/abhinavrastogi-harness/44f7e8858ddc1939f3f968167fd5c871

Reply:
Done